src/integer.rs

```diff
@@ -52,7 +45,10 @@ pub trait PrimitiveInt: EncodableKey {}
 
 impl PrimitiveInt for u32 {}
 
-impl<T> EncodableKey for T where T: Serialize {
+impl<T> EncodableKey for T
+where
+    T: Serialize,
+{
```

Comment:
In general, I like the way rustfmt formats *where* clauses, and it's reasonable to do that consistently, although I might make an exception for very simple declarations like this one, if it were possible to do so (which it doesn't appear to be).